### PR TITLE
feat: gke-defaults package

### PR DESCRIPTION
**Pull request by davelanglois-ssc**
Landing zone v2 subpackage.
Depends on package `gke-setup`.
Requires project-id-tier3 namespace.

Deploy this package once per service project.

A package to deploy common GKE resources. 
- include roles granted to a user or group for visualizing GKE.
- ip reservation for an external global load balancers
- ssl certificate (requires the alpha CRD for now)
- dns record

This package is part of the solution for #479

### CODE REVIEW

**Review by fmichaelobrien — COMMENTED** [review]
Reviewing over the next 48h - I will also test this overlay package on one of my lzs

**Review by fmichaelobrien — DISMISSED** [review]
LGTM

**Review by alaincormier-ssc — DISMISSED** [review]

**Review by cartyc — CHANGES_REQUESTED** [review]

**Review by lucstjean-ssc — APPROVED** [review]
/lgtm

File: solutions/gke/configconnector/gke-defaults/README.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+| certificate-id   |                  12345678 | int   |     1 |
+| certificate-name | certificate-name          | str   |     3 |
+| client-name      | client1                   | str   |     6 |
+| domains          | [example.com]             | array |     1 |
+| gateway-name     | gateway-name              | str   |     2 |
+| host-project-id  | host-project-12345        | str   |     0 |
+| project-id       | project-12345             | str   |    10 |
+| team-gkeviewer   | group:client1@example.com | str   |     6 |
+
+## Sub-packages
```
[the remark this answers is not in the document]
Comment (davelanglois-ssc) [reply solutions/gke/configconnector/gke-defaults/README.md:34]: added with latest commit

File: solutions/gke/configconnector/gke-defaults/gateway-setup/ssl-certificate/ssl.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#########
+# Google Managed Classic SSL certificate
+# Warning! requires the alpha resource loaded in the config controller
+# https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/crds/compute_v1alpha1_computemanagedsslcertificate.yaml
+# TODO: fix error message : Update call failed: error applying desired state: summary: doesn't support update
+apiVersion: compute.cnrm.cloud.google.com/v1alpha1
```
[the remark this answers is not in the document]
Comment (davelanglois-ssc) [reply solutions/gke/configconnector/gke-defaults/gateway-setup/ssl-certificate/ssl.yaml:19]: added with latest commit. please resolve if satified

**Review by cartyc — APPROVED** [review]

File: solutions/gke/configconnector/gke-defaults/gateway-setup/ssl-certificate/ssl.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (cartyc) [reply solutions/gke/configconnector/gke-defaults/gateway-setup/ssl-certificate/ssl.yaml:19]: Looks great! Thanks Dave :)

**Pull request merged**